loadgen/scenario.go

```diff
@@ -199,33 +197,6 @@ func (r *Run) ExecuteKitchenSinkWorkflow(ctx context.Context, options *KitchenSi
 		return fmt.Errorf("failed to start kitchen sink workflow: %w", err)
 	}
 
-	// Ensure custom search attributes are registered
-	_, err = r.Client.OperatorService().AddSearchAttributes(ctx, &operatorservice.AddSearchAttributesRequest{
-		SearchAttributes: map[string]enums.IndexedValueType{
-			"KS_Keyword": enums.INDEXED_VALUE_TYPE_KEYWORD,
-			"KS_Int":     enums.INDEXED_VALUE_TYPE_INT,
-		},
-		Namespace: r.Namespace,
-	})
-
-	// Throw an error if the attributes could not be registered, but ignore already exists errs
-	alreadyExistsStrings := []string{
-		"already exists",
-		"attributes mapping unavailble",
-	}
-	if err != nil {
-		isAlreadyExistsErr := false
-		for _, s := range alreadyExistsStrings {
-			if strings.Contains(err.Error(), s) {
-				isAlreadyExistsErr = true
-				break
-			}
-		}
-		if !isAlreadyExistsErr {
-			return fmt.Errorf("failed to register search attributes: %w", err)
-		}
-	}
-
```

Comment:
I don't have the necessary context here. This was added by @Sushisource in https://github.com/temporalio/omes/pull/41. Is it not needed for anything?

Reply:
Discussed internally. The plan is for clients of omes such as cicd to do this.

Reply:
We still need some way for this to happen automatically so that when developers come along and want to just run omes than they don't have to interpret why things are failing.

This should be automatically done _somewhere_ in this codebase still. It doesn't have to be here, but I don't want to merge this PR without doing it somewhere else, or, at the bare minimum, documenting the necessity

Reply:
That works for me.

Reply:
wait, but, you've just merged it without those things :sweat_smile:

Reply:
Yeah, I was hoping that at least the scenario requirements would be documented (though would prefer a bootstrapping command)